cli/command_server_start.go

```diff
@@ -217,25 +218,28 @@ func (c *commandServerStart) run(ctx context.Context) error {
 			return errors.Wrap(httpServer.Close(), "close")
 		}
 
+		rep := srv.GetRepository()
+		if rep != nil {
+			//nolint:errcheck
+			rep.Close(ctx)
```

Comment:
I discussed with @aaron-kasten offline, just leaving a note here. This is probably going to race with the async init setup for openRepository() before this.